Gulpfile.coffee

```diff
@@ -190,5 +212,34 @@ gulp.task 'serve', ['watch'], ->
                 res.setHeader('Content-Type','image/svg+xml');
             next()
     ]
-
   connect.server(config)
+
+gulp.task '_tdd', (done) ->
+  buildAndTest(true, done)
+  return # Since this is async
+
+
+# -----------------------------------------------------------------------
+#  Public Tasks
```

Comment:
Mind if I split the files in a separate PR? I'm updating #112 with commits in this PR